### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by bitfriend**
- Rename conversation to convo simply
- Remove then from async operation on flutter side
- Renamed `app/lib/features/chat/widgets/conversation_list.dart` to `convo_list.dart`
- Renamed `native/acter/src/api/conversation.rs` to `convo.rs`

### CODE REVIEW

File: app/lib/features/chat/pages/profile_page.dart [hunk 1]
```diff
@@ -180,16 +180,12 @@ class ProfilePage extends ConsumerWidget {
                         buildRequestsAndInvites(context),
```
Comment (gtalha07) [inline app/lib/features/chat/pages/profile_page.dart:180]: It seems like we still have widget methods residing in code structures, might take a look on that maybe separately, extracting them to widgets if possible to adhere to performance patterns.

**Review by gtalha07 — APPROVED** [review]
LGTM!

File: app/lib/features/chat/pages/profile_page.dart (hunk 1 above)
Comment (bitfriend) [reply app/lib/features/chat/pages/profile_page.dart:180]: Okay, will fix them.

**Comment by gnunicorn** [comment]
@bitfriend mind resolving the conflict?!?

**Pull request merged**